modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewPage.kt

```diff
@@ -93,10 +92,16 @@ private fun WhatsNewPageLoaded(
         contentAlignment = Alignment.Center,
         modifier = Modifier
             .background(Color.Black.copy(alpha = scrimAlpha))
-            .clickable(
-                indication = null,
-                interactionSource = remember { MutableInteractionSource() },
-                onClick = { if (!state.fullModel) performClose() },
+            .then(
+                if (!state.fullModel) {
+                    Modifier.clickable(
+                        indication = null,
+                        interactionSource = remember { MutableInteractionSource() },
+                        onClick = { performClose() },
+                    )
+                } else {
+                    Modifier
+                },
```

Comment:
This is to fix an accessibility issue in full mode. We were selecting the whole screen when Talkback is ON, which was causing an issue that we could not select individual components